setup.py

```diff
@@ -49,8 +49,8 @@
         ],
         "blendsearch": ["flaml[blendsearch]"],
         "mathchat": ["sympy", "pydantic==1.10.9", "wolframalpha"],
-        "retrievechat": ["chromadb", "sentence_transformers", "pypdf", "ipython"],
-        "teachable": ["chromadb"],
+        "retrievechat": ["chromadb", "lancedb", "sentence_transformers", "pypdf", "ipython"],
+        "teachable": ["chromadb", "lancedb"],
```

Comment:
It would be better to have only one or even no db here. We can discuss about it later.